go/vt/tabletserver/tabletserver.go

```diff
@@ -571,12 +576,15 @@ func (tsv *TabletServer) isMySQLReachable() bool {
 
 // ReloadSchema reloads the schema.
 func (tsv *TabletServer) ReloadSchema(ctx context.Context) error {
-	defer logError(tsv.qe.queryServiceStats)
-	return tsv.qe.schemaInfo.Reload(ctx)
+	tsv.qe.schemaInfo.ticks.Trigger()
+	return nil
```

Comment:
Doesn't this need to be synchronous, for the wrangler tests to not flake, at least?